### PR TITLE
Project toml path

### PR DESCRIPTION
**Pull request by tylerphelan**

### CODE REVIEW

File: docs/build.md [hunk 1]
```diff
@@ -43,6 +44,7 @@ spec:
 - `source`: The source location that will be the input to the build. See the [Source Configuration](#source-config) section below.
 - `cacheName`: Optional name of a persistent volume claim to used for a build cache across builds.
 - `env`: Optional list of build time environment variables.
+- `projectDescriptorPath`: Path to the [project descriptor file](https://buildpacks.io/docs/reference/config/project-descriptor/) relative to source root dir or `subPath`. If unset, kpack will look for `project.toml` at the root dir.
```
Comment (samj1912) [inline docs/build.md:47]: Should we clarify what root dir means here? It should look at the subpath project.toml if subpath is specified right?

**Review by samj1912 — COMMENTED** [review]
lgtm, minor comment on docs

**Review by tomkennedy513 — APPROVED** [review]

File: pkg/apis/build/v1alpha2/build_types.go [hunk 2]
```diff
@@ -63,10 +63,11 @@ type BuildSpec struct {
 	// +listType
 	Bindings Bindings `json:"bindings,omitempty"`
 	// +listType
-	Env       []corev1.EnvVar             `json:"env,omitempty"`
-	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
-	LastBuild *LastBuild                  `json:"lastBuild,omitempty"`
-	Notary    *NotaryConfig               `json:"notary,omitempty"`
+	Env                   []corev1.EnvVar             `json:"env,omitempty"`
+	ProjectDescriptorPath string                      `json:"projectDescriptorPath,omitempty"`
```
Comment (matthewmcnew) [inline pkg/apis/build/v1alpha2/build_types.go:67]: Should this be `descriptorPath` to match the pack flag of `--descriptor`?

Comment (tylerphelan) [reply pkg/apis/build/v1alpha2/build_types.go]: going with `projectDescriptorPath` to disambiguate from a kp import descriptor

**Pull request merged**